enterprise/config/premium_installation_config.yml

```
@@ -1,18 +1,18 @@
 # ------- Branding Related Config ------- #
-- name: INSTALLATION_NAME
-  value: 'Chatwoot'
+ - name: INSTALLATION_NAME
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 1

(indentation)